dev/create-release/spark-rm/Dockerfile

```diff
@@ -42,7 +42,7 @@ ARG APT_INSTALL="apt-get install --no-install-recommends -y"
 #   We should use the latest Sphinx version once this is fixed.
 # TODO(SPARK-35375): Jinja2 3.0.0+ causes error when building with Sphinx.
 #   See also https://issues.apache.org/jira/browse/SPARK-35375.
-ARG PIP_PKGS="sphinx==3.0.4 mkdocs==1.1.2 numpy==1.20.3 pydata_sphinx_theme==0.8.0 ipython==7.19.0 nbsphinx==0.8.0 numpydoc==1.1.0 jinja2==2.11.3 twine==3.4.1 sphinx-plotly-directive==0.1.3 sphinx-copybutton==0.5.2 pandas==1.5.3 pyarrow==3.0.0 plotly==5.4.0 markupsafe==2.0.1 docutils<0.17 grpcio==1.56.0 protobuf==4.21.6 grpcio-status==1.56.0 googleapis-common-protos==1.56.4"
```

Comment:
I'm not sure if we still need to specify the version from `Dockerfile` or not ?

Reply:
nah let's pin the version for this one

Reply:
So, is this removed in this PR?

Reply:
It seems that `sphinx-copybutton` is accidentally removed in this file. Did I understand correctly, @itholic ?

Reply:
Yeah, I thought we can remove the package here if we don't need to pin the specific version. Let me revert the change.